.travis.yml

```diff
@@ -0,0 +1,29 @@
+language: ruby
+before_install:
+    - brew update
+install:
+    - mkdir -p $(brew --repo)/Library/Taps/travis
+    - ln -s $PWD $(brew --repo)/Library/Taps/travis/homebrew-moira
+    - brew tap --repair
+script:
+    - brew audit -v $PACKAGE
+    - brew install -v $PACKAGE
+    - brew test -v $PACKAGE    
```

Comment:
Errant whitespace at the end of line here.

Reply:
Fixed in 4ade5f4